examples/efficientdet/efficientnet_builder.py

```diff
@@ -0,0 +1,30 @@
+import tensorflow as tf
+import efficientnet.tfkeras as efficientnet
+
+
+def build_backbone(config):
+    """
+    Build backbone model.
+    # Arguments
+        config: Configuration of the EfficientDet model.
+    # Returns
+        EfficientNet model with intermediate feature levels.
+    """
+
+    backbone_name = config["backbone_name"]
+    weight = config["backbone_weight"]
+
+    efficientnet_class = getattr(efficientnet, backbone_name)
+    backbone = efficientnet_class(weights=weight, include_top=False)
+    backbone_layers = backbone.layers
+    features = []
+    for layer, nlayer in zip(backbone_layers[:-1], backbone_layers[1:]):
```

Comment:
nlayer is num_layer or layer_arg?

Reply:
It is actually, a way to filter the activation output before depth separable conv layers of EfficientNet. So layer, nlayer stands for layer, next_layer. I made it explicit in the upcoming push.